snapshots/artificial_intelligence/2025-11-07/energy_ai_iea.py

```diff
@@ -0,0 +1,38 @@
+"""Script to create a snapshot of dataset 'Energy and AI (IEA, 2025)'.
```

Comment:
You could copy these instructions as commented lines into the companion .dvc file.
Then you can simply remove this .py file, and execute the snapshot with:
```
etls artificial_intelligence/2025-11-07/energy_ai_iea --path-to-file ...
```